src/View/html/Settings/system_status.php

```diff
@@ -99,11 +129,13 @@
 				<noscript><?php esc_html_e( 'You need JavaScript enabled to perform this check.', 'gravity-forms-pdf-extended' ); ?></noscript>
 
 				<div id="gfpdf-direct-pdf-check-protected" style="display: none">
-					<?php esc_html_e( 'Protected', 'gravity-forms-pdf-extended' ); ?> <span class="fa fa-check-circle"></span>
+					<?php esc_html_e( 'Protected', 'gravity-forms-pdf-extended' ); ?>
```

Comment:
Best not to make changes that aren't relevant to your PR. I know you ran the code formatted in PHPStorm and this was automatic, but that should be handled as a separate PR. Please revert changes that aren't relevant.